### ISSUE TITLE
cannot add buttons to keyboard with emoji as bytes

### ISSUE THREAD

**Issue by Yolley** [eternnoir/pyTelegramBotAPI#265]
Please answer these questions before submitting your issue. Thanks!

1. What version of pyTelegramBotAPI are you using?
2.2.3

2. What version of python are you using?
3.5.2

I have a message_handler:
```
@bot.message_handler(commands=['vote'])
def voter(message):
	markup = telebot.types.ReplyKeyboardMarkup(one_time_keyboard=True, selective=True)
	markup.add(b'\xF0\x9F\x94\xA5')
	markup.add(b'\xF0\x9F\x93\xB7')
	bot.reply_to(message, 'Choose', reply_markup=markup)
```

Which tries to make a custom keyboard with emoji's. I represent them as bytes and it works well, when i just send them with send_message.
But when i try to add a button to keyboard, i get an error:
```
File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/telebot/types.py", line 610, in add
    row.append(button.to_dic())
AttributeError: 'bytes' object has no attribute 'to_dic'
```

I think, it's a bug, cuz if i add to 'add' function something like this:
```
            elif isinstance(button, bytes):
                row.append({'text': button.decode('utf-8')})
```

It works well, and buttons add successfully to keyboard.
Is it going to be fixed?

Thank you!

**Comment by Kondra007** [comment]
Have you tried copy-pasting emojis into your IDE directly? Works fine for me, since emoji = Unicode character

**Comment by Yolley** [comment]
Actually this idea didn't come to my mind. Tried it, and yes, it worked, thank you!

Before trying to represent them as bytes, I tried to past emoji's with unicode, but i didn't copy the emoji itself, i copied the code from there http://apps.timwhitlock.info/emoji/tables/unicode, and every emoji looked like this
![emoji](https://cloud.githubusercontent.com/assets/13859030/20940881/5c33f22e-bc05-11e6-8576-5208a56ce647.png)
So that's why i thought, that i should try to past them as bytes.
